Wokoro/way_farer_test
# #167275623 Implentation for viewing trips

#### What does this PR do?
PR enable users view all available trips

#### Description of Task to be completed?
Users of the platform should be given ability to view all available trips on the platform when they navigate to `api/v1/trips`.

#### How should this be manually tested?
* Login into the platform with a registered email and password
* Navigate to `api/v1/trips`, to see all trips

#### Any background context you want to provide?
none

#### What are the relevant pivotal tracker stories?
[#167275623](https://www.pivotaltracker.com/story/show/167275623) [#167275625](https://www.pivotaltracker.com/story/show/167275625)

#### Screenshots (if appropriate)
none

#### Questions:
none
